### PR TITLE
Update Add Product Guide

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

Update add products guide.

#### Rationale:
Make life easier for developers.

### CODE REVIEW

File: docs/manual/developer/03_creating_content.md [hunk 1]
```diff
@@ -411,8 +411,8 @@ MAKEFILE_ID_TO_PRODUCT_MAP = {
 
 
 5. Create a new file in the product directory called `CMakeLists.txt`:
-```
```
Comment (vojtapolasek) [inline docs/manual/developer/03_creating_content.md:414]: Not sure why this line is changed... I think it should stay the same.

File: docs/manual/developer/03_creating_content.md [hunk 2]
```diff
@@ -411,8 +411,8 @@ MAKEFILE_ID_TO_PRODUCT_MAP = {
...
-cat << EOF >> $NEW_PRODUCT/CMakeLists.txt
```
Comment (vojtapolasek) [inline docs/manual/developer/03_creating_content.md:415]: I see multiple instances of such changes in this PR. But I think they are not correct - this command is using Bash heredoc to insert contents into a file and the change you propose breaks it.
Heredoc is denoted with <<.
https://www.howtogeek.com/719058/how-to-use-here-documents-in-bash-on-linux/

File: docs/manual/developer/03_creating_content.md [hunk 3]
```diff
@@ -437,7 +437,7 @@ pkg_manager: "yum"
```
Comment (vojtapolasek) [inline docs/manual/developer/03_creating_content.md:437]: Additional .. should be added here as well.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello @Mab879  and thank you for this update, it is really helpful because I can imagine it can cause pain for those who try to add new products.
I think that some changes are not needed. Please see my comments.

File: docs/manual/developer/03_creating_content.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Mab879) [reply docs/manual/developer/03_creating_content.md:437]: The update you requested should be in now.

**Review by vojtapolasek — APPROVED** [review]
Looks good now. Thank you.

**Pull request merged**